src/pip/_internal/wheel.py

```diff
@@ -885,7 +892,10 @@ def _build_one_inside_env(self, req, output_dir, python_tag=None):
                 wheel_name = os.path.basename(wheel_path)
                 dest_path = os.path.join(output_dir, wheel_name)
                 try:
+                    wheel_hash, _ = hash_file(wheel_path)
                     shutil.move(wheel_path, dest_path)
+                    logger.info('Finished: %s sha256=%s',
+                                wheel_name, wheel_hash.hexdigest())
```

Comment:
The other log messages in your post have the form "Running setup.py bdist_wheel for simple: finished ..." So maybe this could be "Finished wheel for \<name>: ..."

Reply:
Also, this `info()` message can be merged with the one right after. Maybe the directory path can go on a separate line like it is now.

Reply:
I guess I don't feel strongly about merging the log messages. However, I'd rather not create any expectation among users (or future maintainers for that matter) that log messages won't or shouldn't change. One reason is that I think we should be open to improving these messages when we can as a matter of UX. Thus, my preference would be for the comment not to be there.

Reply:
> The line will become quite long with the full path too; 

By the way, to clarify, my suggestion was for the second log message to be in the same log message as the first but still be on a separate line (i.e. a newline character would go in the message). Thus, the lengths of lines needn't increase, and in fact the same greps could even work if the text was kept the same.

Reply:
> Thus, my preference would be for the comment not to be there.

Removed with https://github.com/pypa/pip/pull/5908/commits/45a6415ef0f7b5a02061eb6a6c0455b0f5d49537

> By the way, to clarify, my suggestion was for the second log message to be in the same log message as the first but still be on a separate line (i.e. a newline character would go in the message).

Ahh ... yes I wasn't sure about multi-line log strings.  This probably comes from some ancient history of mine where I was dealing with ``syslog()`` and ``/dev/msg`` a lot where multi-line strings are not handled well at all :)  Again if you feel strongly happy to convert it to a suggested format